src/test/mochitest/head.js

```diff
@@ -41,6 +41,10 @@ Services.prefs.setBoolPref("devtools.debugger.new-debugger-frontend", true);
 Services.prefs.clearUserPref("devtools.debugger.tabs")
 Services.prefs.clearUserPref("devtools.debugger.pending-selected-location")
 
+this.gBrowser = gBrowser;
+this.Services = Services;
+this.EXAMPLE_URL = EXAMPLE_URL;
```

Comment:
my fear here is that it's a memory leak. Also, ugly... but perhaps we can do something in another pass when we're cleaning up